README.md

```diff
@@ -1,20 +1,239 @@
+- example body:
+```
+{
+	"id": "INT: on put only",
+	"purchase_date": "TEXT NOT NULL",
+	"decommission_date": "TEXT",
+	"serial_number": "TEXT NOT NULL"
+}
+```
+
+### **Training-Programs**
+- http methods supported: GET, POST, PUT, DELETE
+> NOTE: deleting a training program will delete the associated entry on the employeeTraining join table
+- example body:
+```
+{
+	"id": "INT: include on PUT only",
+	"start_date": "TEXT",
+	"end_date": "TEXT",
+	"max_attendance": "INT",
+	"title": "TEXT"
+}
+```
+
+### **Users**
+- http methods supported: GET, POST, PUT
+- example body:
+```
+{
+	"id": "INT: put only",
+	"first_name": "TEXT",
+	"last_name": "TEXT",
+	"account_created_date": "TEXT NOT NULL",
+	"last_login_date": "TEXT NOT NULL",
+	"street_address": "TEXT",
+	"city_address": "TEXT",
+	"state_code": "TEXT",
+	"zip_code": "TEXT"
+}
+```
+
+### **Product Types**
+- http methods supported: GET, POST, PUT, DELETE
+NOTE: you cannot delete a product type if there are products associated with that product type
+- example body:
+```
+{
+	"id": "INT: PUT only",
+	"name": "TEXT"
+}
+```
+
+### **Products**
+- http methods supported: GET, POST, PUT, DELETE
+> NOTE: deleting a product will delete the associated entry on the orderProduct join table
+- example body:
+```
+{
+	"id": "INT: PUT only",
+	"product_type_id": "INTEGER",
+	"price": "REAL",
+	"title": "TEXT",
+	"description": "TEXT",
+	"original_quantity": "INTEGER",
+	"seller_user_id": "INTEGER"
+}
+```
+
+### **Orders**
+- http methods supported: GET, POST, PUT, DELETE
+> NOTE: deleting an order will delete the associated entry/entries on the orderProduct join table
+- example body:
+```
+{
+	"id": "INT: PUT only",
+	"customer_user_id": "INTEGER",
+	"payment_type_id": "INTEGER",
+	"order_date": "TEXT"
+}
+```
+
+### **Detailed Orders**
+- http methods supported: GET, PUT, DELETE
+> NOTE: ```api/v1/orders/:id/products```
+> NOTE: deleting a detailed order will delete the associated entry/entries on the orderProduct join table
+- example body:
+```
+{
+	"orderBody(above)": "GET only",
+	"product_id": "INT: required on POST/PUT/DELETE",
+	"quantity": "INT: optional on POST/PUT/DELETE, 1 by default"
+}
+```
+
+
+### **Payment Types**
+- http methods supported: GET, POST, PUT, DELETE
+> NOTE: you cannot delete a payment type if there are orders associated with that payment type
+- example body:
+```
+{
+	"id": "INT: PUT only",
+	"customer_user_id": "INTEGER",
+	"type": "TEXT",
+	"account_number": "INTEGER"
+}
+```
+
+### **Inactive Customers**
+- http methods supported: GET
+- route: ```api/v1/users/?active=false```
+
 
 ## Third Party Libraries
-- Express
+- [express](https://www.npmjs.com/package/express)
+- [sqlite3](https://www.npmjs.com/package/sqlite3)
+- [nodemon](https://www.npmjs.com/package/nodemon)
+- [dotenv](https://www.npmjs.com/package/dotenv)
+- [body-parser-json](https://www.npmjs.com/package/body-parser-json)
+
+
+## Credits
```

Comment:
Seems sub headings are bolder than the main heading Credits. Does it look weird?